mobile/Navigation/AppNavigation.jsx

```diff
@@ -28,24 +25,40 @@ import EditPasswordScreen from '../Screens/EditPasswordScreen';
 import SignUpScreen from '../Screens/SignUpScreen';
 import LogInScreen from '../Screens/LogInScreen';
 import PasswordResetScreen from '../Screens/PasswordResetScreen';
-// import SavedScreen from '../Screens/SavedScreen';
 import TripleFlipScreen from '../Screens/WritingActivities/TripleFlipScreen';
 import HistoryScreen from '../Screens/HistoryScreen';
 
 const StoryStarterStack = createNativeStackNavigator();
 
 function StoryStarterStackScreen() {
   return (
-    <StoryStarterStack.Navigator initialRouteName="Story Starters">
+    <StoryStarterStack.Navigator
+      initialRouteName="Story Starters"
+      screenOptions={{ headerTransparent: true, headerTitle: '' }}
+      // Currently back button appears blue instead of white
+      // Can configure back button and header, more info: https://reactnavigation.org/docs/headers/
+    >
       <StoryStarterStack.Screen
         name="Story Starter Stack"
         component={StoryStarterScreen}
-        options={{ headerShown: false, title: 'Story Starters' }}
+        options={{ headerShown: false }}
+      />
+      <StoryStarterStack.Screen
+        name="Objects"
+        children={() => (<StoryStarterComponent title="OBJECT" route="item" textColor="#7BAC8A" />)}
```

Comment:
Try to avoid passing children as props!  Instead, it's better practice to nest the child components between the opening and closing tags of the parent component. Something like this should work:
```
      <StoryStarterStack.Screen
        name="Objects"
      >
        {() => (<StoryStarterComponent title="OBJECT" route="item" textColor="#7BAC8A" />)}
      </StoryStarterStack.Screen>
```

Reply:
fixed!